.mypy.ini

```diff
@@ -6,11 +6,6 @@ exclude = .*plugins\/.*
 ; ONLY ignores files encountered while recursively discovering files to check.
 ; Without this option, mypy will ALWAYS include 'atomic_reactor/plugins/post_rpmqa.py'
 ; Yes it did take me ~1.5 hrs to figure this out.
```

Comment:
This comment is probably unneeded now (but should be tested)

Reply:
thanks, removed